src/parse/ris_live.go

```diff
@@ -79,44 +80,49 @@ type RisLiveMessage struct {
 }
 
 // connects to ris live, starts go routine receiverHandler, manages connection and subscription
-func ParseRisLiveData(msgChannel chan common.Message, config *config.Configuration) error {
+func ParseRisLiveData(msgChannel chan common.Message, config *config.Configuration) {
```

Comment:
I think this is not being called in the committed code?

Reply:
addressed in my next PR, where main.go will check if config wants live vs static analysis and call corresponding function